realm/kotlin-extensions/src/androidTest/kotlin/io/realm/KotlinRealmTests.kt

```diff
@@ -61,4 +66,88 @@ class KotlinRealmTests {
     fun where() {
         assertEquals(0, realm.where<SimpleClass>().count())
     }
-}
+
+    @Test
+    fun where_getsResult() {
```

Comment:
Not entirely sure what this tests? Is it `where` ? In which case `assertEquals(0, realm.where<PrimaryKeyClass>().count())` would be enough?

If it is more, then all of that is tested on the Java side and doesn't need to be tested from Kotlin I think?

Reply:
My thought is that it's testing that the extension where() method can (setup a RealmQuery) to get data from Realm.  It just seemed like a deeper/more thorough test that the simple where() test method. 

Perhaps too much?  I can remove.   Let me know

Reply:
`where()` returns a `RealmQuery` which is already pretty well tested from Java and the Kotlin API doesn't change anything once you have the `RealmQuery`, so in the context of Kotlin it is probably a bit too much. I would probably remove it, also to keep the cognitive load down.

Reply:
👍 will do